File: Lab3/pzpi-22-8-hmyrak-mykhailo-lab3/src/types/book.ts
export interface Book {
  bookId: number;
  title: string;
  isbn: string;
  pages: number;
  publish: string;
  categoryId: number;
  class?: string;
  lang?: string;
  year: number;
  itemsCount: number;
  availableItemsCount: number;
}

File: Lab5/pzpi-22-8-hmyrak-mykhailo-lab5/mobile-client/app/src/main/java/com/example/apz_library/data/models/Book.kt
package com.example.apz_library.data.models

data class Book(
    val bookId: Int,
    val title: String,
    val isbn: String?,
    val pages: Int?,
    val publish: String?,
    val categoryId: Int?,
    val class_: String?,
    val lang: String?,
    val year: Int?,
    val itemsCount: Int?,
    val availableItemsCount: Int?
)